angular/src/app/components/release-management/release-detail/release-detail.component.ts

```diff
@@ -0,0 +1,129 @@
+import { Component, OnInit, Input, Output, EventEmitter, ElementRef, ViewChild, AfterViewInit } from '@angular/core';
+import { Release } from 'src/app/model/release';
+import * as BpmnJS from 'bpmn-js/dist/bpmn-viewer.production.min.js';
+import { CamundaApiConnectorService } from 'src/app/services/releasemanagement/camunda-api-connector.service';
+import { MatDialog } from '@angular/material/dialog';
+import { EditTaskComponent } from "src/app/components/release-management/edit-task/edit-task.component";
+import { CamundaTask } from 'src/app/model/camunda-task';
+import { ReleaseVariable } from 'src/app/model/release-variable';
+
+@Component({
+  selector: 'app-release-detail',
+  templateUrl: './release-detail.component.html',
+  styleUrls: ['./release-detail.component.css']
+})
+export class ReleaseDetailComponent implements OnInit, AfterViewInit {
+
+  @ViewChild('bpmncontainer', { static: true }) private bpmnContainer: ElementRef;
+
+  @Input() release: Release = null;
+  @Output() emitRelease = new EventEmitter<Release>()
+
+  public isLoading: boolean = false;
+
+  constructor(private camunda: CamundaApiConnectorService, public dialog: MatDialog) { }
+
+  ngOnInit(): void {
+    this.loadReleaseTasks()
+  }
+
+  ngAfterViewInit(): void {
+    //Called after ngAfterContentInit when the component's view has been initialized. Applies to components only.
+    //Add 'implements AfterViewInit' to the class.
+    this.loadBpmnDiagram()
+  }
+
+  loadReleaseTasks() {
+    this.camunda.getReleaseTasksByInstanceId(this.release.processInstanceId)
+      .subscribe(tasks => {
+        this.release.tasks = tasks;
+      })
+  }
+
+  updateReleaseVariables() {
+    this.isLoading = true;
+
+    this.camunda.getAllReleaseVariablesByInstanceId(this.release.processInstanceId)
+      .subscribe(
+        updatedVariables => {
+        this.release.variables = updatedVariables;
+        this.isLoading = false;
+        },
+        error => {
+          console.warn("Keine Variablen gefunden, Instanz womöglich gelöscht", error)
```

Comment:
isLoading stays true

Reply:
fixed in latest [commit ](https://github.com/Sultanow/js_menus/pull/60/commits/158507961befce32f22c37648fd6f434a538cbca)